src/main/app/src/app/taken/taak-view/taak-view.component.html

```diff
@@ -24,11 +24,13 @@
                     </div>
 
                     <div [fxFlex]="100/3" fxFlex.lt-md="50">
-                        <zac-edit-groep [formField]="editFormFields.get('groep')"
+                        <zac-edit-groep [readonly]="isAfgerond()"
+                                        [formField]="editFormFields.get('groep')"
                                         (onSave)="editGroep($event)"></zac-edit-groep>
-                        <zac-edit-behandelaar [formField]="editFormFields.get('behandelaar')"
+                        <zac-edit-behandelaar [readonly]="isAfgerond()"
+                                              [formField]="editFormFields.get('behandelaar')"
                                               (onSave)="editBehandelaar($event)"
-                                              [showAssignToMe]="taak.rechten[taakRechten.TOEKENNEN_AAN_MIJ]"
+                                              [showAssignToMe]="showAssignToMe()"
```

Comment:
Wil je niet die constructie houden? Ik meende dat we een ander soort class gingen introduceren hiervoor? Iets met een andere naam dan rechten